components/browser/awesomebar/src/test/java/mozilla/components/browser/awesomebar/BrowserAwesomeBarTest.kt

```diff
@@ -385,6 +385,29 @@ class BrowserAwesomeBarTest {
         assertEquals(1, awesomeBar.getUniqueSuggestionId(AwesomeBar.Suggestion(id = "1", score = 0, provider = provider)))
     }
 
+    @Test
+    fun `provider exists for the awesome bar`() {
+        val awesomeBar = BrowserAwesomeBar(testContext)
+        val provider1 = mockProvider()
+        `when`(provider1.id).thenReturn("1")
+
+        awesomeBar.addProviders(provider1)
+        assertEquals(true, awesomeBar.containsProvider(provider1))
+    }
+
+    @Test
+    fun `provider does not exist for the awesome bar`() {
```

Comment:
Since the two tests are essentially the same, let's combine them into one, e.g:

```Kotlin
@Test
fun `containsProvider - checks if provider exists`() {
val awesomeBar = BrowserAwesomeBar(testContext)
    val provider1 = mockProvider()
    `when`(provider1.id).thenReturn("1")

    val provider2 = mockProvider()
    `when`(provider2.id).thenReturn("2")

    awesomeBar.addProviders(provider1)
    assertTrue(awesomeBar.containsProvider(provider1))
    assertFalse(awesomeBar.containsProvider(provider2))
}
```